data/operators/amenity/car_sharing.json

```diff
@@ -479,21 +480,33 @@
       }
     },
     {
-      "displayName": "RTTB",
-      "id": "rttb-672a50",
-      "locationSet": {"include": ["fx"]},
+      "displayName": "Scouter",
+      "id": "scouter-088af7",
+      "locationSet": {"include": ["de"]},
       "tags": {
         "amenity": "car_sharing",
-        "operator": "RTTB"
+        "operator": "Scouter"
       }
     },
     {
-      "displayName": "Scouter",
-      "id": "scouter-088af7",
-      "locationSet": {"include": ["de"]},
+      "displayName": "Share Now",
+      "id": "sharenow-6a8c71",
+      "locationSet": {
+        "include": [
+          "at",
+          "ca",
+          "de",
+          "dk",
+          "fx",
```

Comment:
`fx` doesn’t appear to be a valid ISO 3166-2 code. Did you mean `fr` for France?

Reply:
Yes, it was for France, I found some other exemples (for example in the same file "Alpes Autopartage" and "Autolib") tagged with `fx` instead of `fr`, I assumed it was similarly to the ISO-3 `FXX` code, only for the Metropolitan France (excluding overseas territories)?

Reply:
Oh yeah `fx` is allowed here, maybe even preferred.
https://ideditor.codes/?code=fx

Country Coder supports it, I guess because ISO 3166 includes it as an exception code:  
https://www.iso.org/obp/ui/#iso:code:3166:FX
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Exceptional_reservations

The complete list of codes supported is 
https://ideditor.codes/
https://github.com/ideditor/country-coder

Reply:
Thanks, I learned something new!